components/AppManager.jsx

```diff
@@ -0,0 +1,31 @@
+import { useDeferredValue, useEffect, useState } from "react";
+
+import {Routes, Route} from 'react-router-dom'
+
+import ArticlesList from './ArticlesList'
+
+// import { useContext } from "react";
+// import { ArticlesContext } from "./context/articlesContext";
+
+
+export default function AppManager () {
+    const [articles, setArticles] = useState()
+
+    useEffect(() => {
+        const fetchAllArticles = async () => {
+            const articleQuery = await fetch(`https://be-nc-news-famm.onrender.com/api/articles`) 
```

Comment:
you might want to consider using axios in your project as it does the json-ifying for you and is a slightly nicer syntax when we start working with params and such. I'd also heavily consider extracting API calls to a API file early on to neaten up your code!